logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/ObserveConversationViewUseCaseImpl.kt

```diff
@@ -0,0 +1,17 @@
+package com.wire.kalium.logic.feature.conversation
+
+import com.wire.kalium.logic.data.conversation.ConversationRepository
+import com.wire.kalium.logic.data.conversation.ConversationView
+import kotlinx.coroutines.flow.Flow
+
+fun interface ObserveConversationViewUseCase {
+    suspend operator fun invoke():  Flow<List<ConversationView>>
+}
+
+internal class ObserveConversationViewUseCaseImpl(
```

Comment:
The fact that it is a `View` in the database doens't mean we should name `ConversationView` all the way up to UseCases.

I think we could call it `ConversationDetails`, because that's what it is replacing, and that's what it is really.

Reply:
Just wanted to diffrentiate it from the preview queries, now after more clean up we can select better namings.
But still it's not a conversationDetail! it's more like a conversationHeader :D

Reply:
The old wasn't called `ConversationDetails` it was called `ConversationListDetails` which still seems like a fitting name.